queue_services/entity-filer/src/entity_filer/filing_processors/cease_receiver.py

```diff
@@ -28,12 +28,13 @@ def process(business: Business, filing: Dict, filing_rec: Filing, filing_meta: F
         return
 
     if parties := cease_receiver_filing.get('parties'):
-        update_parties(parties)
+        update_parties(parties, business)
 
 
-def update_parties(parties: dict):
+def update_parties(parties: dict, business: Business):
     """Cease receiver party role."""
     end_date_time = datetime.datetime.utcnow()
     for party in parties:
-        party_role = PartyRole.find_by_internal_id(internal_id=party.get('officer').get('id'))
```

Comment:
earlier logic was wrong (checking role by internal id). Changed it ot look at part id